Lib/fontbakery/specifications/shared_conditions.py

```diff
@@ -66,7 +66,7 @@ def monospace_stats(ttFont):
       most_common_width = width
   # if more than 80% of glyphs have the same width
   # then the font is very likely considered to be monospaced
-  seems_monospaced = occurrences > 0.80 * len(glyphs)
+  seems_monospaced = occurrences > 0.80 * len(glyphs.keys())
```

Comment:
maybe the fonttools TTGlyphSet class should grow a `__len__` method like the built-in `dict`, so one would not need to do `len(glyphs.keys())`

Reply:
Sure! https://github.com/fonttools/fonttools/commit/4a16d1839d86eb8be12c6bb546c1ea8bf4ebfcb1